specs/spec_helper.rb

```diff
@@ -0,0 +1,111 @@
+# spec_helper.rb
+
+# to do: figure out Rake
+
+# Add simplecov
+require "minitest"
+require "minitest/autorun"
+require "minitest/reporters"
+
+Minitest::Reporters.use!
+Minitest::Reporters::SpecReporter.new
+
+require_relative "../lib/hotel.rb"
+
+# initialize a hotel system
+describe "HotelSystem" do
```

Comment:
Typically we create separate `filename_spec.rb` files in the `specs` folder and use `require_relative` to require the spec helper and run the tests with the `rake` command.  

I can walk you through this later when you start.